install/update_955_956.php

```diff
@@ -42,21 +42,52 @@ function update955to956() {
    $current_config   = Config::getConfigurationValues('core');
    $updateresult     = true;
    $ADDTODISPLAYPREF = [];
+   $init_date = false;
 
    //TRANS: %s is the number of new version
    $migration->displayTitle(sprintf(__('Update to %s'), '9.5.6'));
    $migration->setVersion('9.5.6');
 
-   //put you migration script here
-
    // Change DC itemtype template_name search option ID from 50 to 61 to prevent duplicate IDs now that those itemtypes have Infocom search options.
    $migration->changeSearchOption(Enclosure::class, 50, 61);
    $migration->changeSearchOption(PassiveDCEquipment::class, 50, 61);
    $migration->changeSearchOption(PDU::class, 50, 61);
    $migration->changeSearchOption(Rack::class, 50, 61);
 
+   /* Add `date` to some glpi_documents_items */
+   if (!$DB->fieldExists('glpi_documents_items', 'date')) {
+      $init_date = true;
+      $migration->addField('glpi_documents_items', 'date', 'timestamp');
+      $migration->addKey('glpi_documents_items', 'date');
+   }
+   /* /Add `date` to glpi_documents_items */
+
    // ************ Keep it at the end **************
    $migration->executeMigration();
 
+   // Must be run after migration is executed so we can access the new fields
```

Comment:
I guess you can use Migration::migrationOneTable  to get rid of this

Reply:
You can use `addPostQuery`, it is even cleaner.

Reply:
Indded :)